### ISSUE TITLE
`CheckedC-Clang-12.0.1-rel1` tag created from `updated_baseline_master_12` instead of `release_12.x`: mistake?

### ISSUE THREAD

**Issue by mattmccutchen-cci** [microsoft/checkedc-clang#1143]
@sulekhark I just noticed that you created [a GitHub release](https://github.com/microsoft/checkedc-clang/releases/tag/CheckedC-Clang-12.0.1-rel1) of the Checked C compiler, but the corresponding tag `CheckedC-Clang-12.0.1-rel1` points to `updated_baseline_master_12` (373b5dc566657e7e7260f8e2d996f713090ec256), which is based on the LLVM 12 branch point without the LLVM 12.0.1 patches from the release branch.  I thought your plan was to include those patches in the release, which would mean the tag should point to `release_12.x` (142a7d1c195ca591eff0e4929e629ea2709fa849).  This would follow the pattern of `CheckedC-Clang-11.1.0-rel1`, which points to the same commit as `release_11.x` (f0367c35f7c38133f23ba64e5d8dbfb6bdfe0987).  I haven't tried to check whether the Windows binaries attached to the new release were actually built from `updated_baseline_master_12` or `release_12.x`.

Assuming you did intend to use `release_12.x`: Overwriting tags in public Git repositories is generally discouraged.  If you were to overwrite the `CheckedC-Clang-12.0.1-rel1` tag now, by default, Git would refuse to propagate the change to any repositories that have already fetched the tag, such as mine (although I may well be the only user who checked so far :slightly_smiling_face:).  Thus, your best option at this point is probably to create a new tag `CheckedC-Clang-12.0.1-rel2` and a new corresponding GitHub release.  If the binaries were already built from 142a7d1c195ca591eff0e4929e629ea2709fa849 and the build process didn't embed the tag name 
`CheckedC-Clang-12.0.1-rel1` in them anywhere, you can probably just rename the files to `rel2` instead of `rel1` and reattach them to the new GitHub release.

**Comment by sulekhark** [comment]
@mattmccutchen-cci , thanks for noticing this! I just confirmed that the release exes are build from the commit hash [142a7d1](https://github.com/microsoft/checkedc-clang/commit/142a7d1c195ca591eff0e4929e629ea2709fa849) off branch `release_12.x`. The tag `CheckedC-Clang-12.0.1-rel1` got inadvertently applied to the `master` branch instead of `release_12.x` branch. I have made a new corrected release with the release tag  `CheckedC-Clang-12.0.1-rel2` as per your (last) suggestion.
